src/clients/stsService.test.js

```
@@ -0,0 +1,121 @@
+import { beforeEach, describe, expect, jest, test } from '@jest/globals';
+import { mockClient } from 'aws-sdk-client-mock';
+import { STSClient, GetCallerIdentityCommand } from '@aws-sdk/client-sts';
```

ESLint

[error] Member 'GetCallerIdentityCommand' of the import declaration should be sorted alphabetically.

(sort-imports)